src/worker.cc

```diff
@@ -408,6 +410,10 @@ void parse_location(valhalla::Location* location,
     // search_filter.exclude_ramp
     location->mutable_search_filter()->set_exclude_ramp(
         rapidjson::get<bool>(*search_filter, "/exclude_ramp", false));
+    auto level =
+        rapidjson::get<float>(*search_filter, "/level", static_cast<float>(baldr::kMaxLevel));
+    if (level != baldr::kMaxLevel)
+      location->mutable_search_filter()->set_level(level);
```

Comment:
need to worry about when person uses a level that is higher than max and also the raw proto input variant of this has to get validated

Reply:
no real need though to clamp since max level is already the numerical limit of `float`, right?